unit-tests/song-services.test.js

```diff
@@ -0,0 +1,64 @@
+const songServiceFunctions = require('../songbank-backend/song-services.js');
+
+var testPlayListTitle = 'Playlist 1'
+
+var testSongTitle = 'Easy On Me'
+
+var testPlaylist = {
+    title: "Test Playlist",
+    songs: [{
+        title: 'Easy On Me',
+        Artist: 'Adele',
+        runtime: 3.45,
+    }]

```

Comment:
When testing the usage of playlists, does the number of songs in the playlist matter? If not, good job only using one song